packages/node_modules/@webex/plugin-meetings/src/roap/index.js

```diff
@@ -245,12 +256,17 @@ export default class Roap extends StatelessWebexPlugin {
       correlationId: meeting.correlationId
     });
 
+    // When reconnecting, it's important that the first roap message being sent out has empty media id.
+    // Normally this is the roap offer, but when TURN discovery is enabled,
+    // then this is the TURN discovery request message
+    const sendEmptyMediaId = reconnect && !meeting.config.experimental.enableTurnDiscovery;
```

Comment:
So now any kind of reconnect , either media release or inactive the client will not send media ID

Reply:
Yeah the assumption is that in all cases it will always go through either `ReconnectionManager.setupPeerConnection()` or a new call to `addMedia()` so we will do TURN discovery again. Is that a correct assumption?